tests/model_creators/generic_creators.py

```diff
@@ -528,7 +528,8 @@ def create_venue_provider(venue: Venue,
                           is_active: bool = True,
                           last_provider_id: int = None,
                           last_sync_date: datetime = None,
-                          venue_id_at_offer_provider: str = None
+                          venue_id_at_offer_provider: str = '123456789',
```

Comment:
```suggestion
                          venue_id_at_offer_provider: str = None,
```
J'ai essayé de garder une cohérence avec le model quand je suis repassé sur toutes ces fonctions.
Comme ce champ peut être `null`, je trouve pertinent de le laisser à `None`.
Quand penses tu ?